app/src/main/java/info/nightscout/android/medtronic/service/MedtronicCnlService.java

```diff
@@ -303,15 +303,15 @@ public void run() {
             cnl0x81 = 0;
 
             try {
-
+/*
```

Comment:
Remove commented code if not necessary

Reply:
It's there for debugging / development use. It invokes the history reading from file rather then from the cnl.

Reply:
No worries.